dandischema/metadata.py

```diff
@@ -123,8 +123,8 @@ def generate_context() -> dict:
 
     for item in models.DigestType:
         fields[item.value] = {"@id": item.value, "@nest": "digest"}
-    fields["Dandiset"] = "dandi:Dandiset"
-    fields["Asset"] = "dandi:Asset"
+    fields["Dandiset"] = f"{models.DANDI_NSKEY}:Dandiset"
+    fields["Asset"] = f"{models.DANDI_NSKEY}:Asset"
```

Comment:
these should not change as those prefixes are not urls just keys in jsonld - they won't resolve. the lower case `dandi` is simply a uri.